src/rfc-8252-http-server.ts

```diff
@@ -84,6 +128,39 @@ export class RFC8252HTTPServer {
     if (!baseUrl) {
       throw new MongoDBOIDCError('Received HTTP request while not listening');
     }
+
+    let isAcceptedOIDCResponse = false;
+    const reject = (
+      info: Omit<
+        RedirectServerRequestInfo & { result: 'rejected' },
+        'req' | 'res' | 'result'
+      >
+    ) => {
+      this.logger.emit('mongodb-oidc-plugin:oidc-callback-rejected', {
+        method: req.method,
+        hasBody,
+        errorCode: info.error ?? 'unknown_error',
```

Comment:
Is `info.error` already just a code string or it is the whole error object here being assigned to the `errorCode`?

Reply:
It's a string: https://github.com/mongodb-js/oidc-plugin/blob/b1115053a7b5c47f847bf819b73d3e5cefb37001/src/api.ts#L187-L188